README.md

```diff
@@ -120,4 +100,12 @@ In addition to the Selenium 2 implicit waiting, the ```AutomationTest``` class e
 [See a working example](https://github.com/ddavison/conductor/blob/master/src/test/java/io/ddavison/conductor/FrameworkTest.java) of what a test script written using this framework might look like.
 
 # Pull requests
-If you have an idea for the framework, fork it and submit a pull-request!
+If you have an idea for the framework, fork it and submit a pull-request for the develop branch!
+
+# Release process
+We follow gitflow branch management [reference graphic](http://nvie.com/posts/a-successful-git-branching-model/). The
+steps to make a new release are therefor:
+1. Create a release branch from the develop branch named `release/x.x.x`
+2. Create a new pull request from the release branch to the master branch
+3. If approved merge release branch into master
+4. Tag the merge (with release notes) in the master branch with `x.x.x` (this will make this version available in jitpack)
```

Comment:
Are we missing a step to merge back into `develop`?

Reply:
Yeah, I wasn't sure how in depth I should get here. I also left out bugfix branches from the release branch.

Would you rather have a PR from the release branch to develop or from master to develop? Effectively it's the same thing. I'm more used to going from master but the graphic shows a merge from the release branch. I'm fine with either.

Reply:
Yeah, I think the way I've worked with it is to branch from develop > release branch > into develop > into master. Then tagging happens in master.

Bugfixes, however, usually go master > bugfix branch > master > tagging master > to develop.

Reply:
Included the merge back into develop instructions. Should be a PR because there should be no direct commits into develop.